index.html

```diff
@@ -96,9 +114,10 @@ <h3 class="card-main-title">Profesional Art Printing Data</h3>
                         </section> 
                         <button type="button" class="card-big-btn" >See project</button>    
                     </section>
-                    <section class="post">
+                    <section class="post post3">
                         <section class="card-contents">
                             <h3 class="card-main-title">Profesional Art Printing Data</h3>
+                            <h3 class="card-main-title1">Website Portfolio </h3>
                             <p class="card-description">A daily selection of privately personalized reads;
                                no accounts or sign-ups required. Has been the industry's standard.
                             </p>
```

Comment:
- [ ] You have done a great job so far, But here is also one important aspect of your project you can still improve on before moving to the next milestone. 😄 👍 👍 
 - - You need to implement the hover state on each of your project cards. By default in the desktop view, the `See Project button` is hidden, but when a user hovers over a project, the `See project button`  should appear at the bottom of the project card.
 - - Currently, the see project button is not visible on your website even when a user hovers or clicks on a project, That's not good for the user experience as users can't see any button to click if they want to explore your project.
 - Screenshot for reference:
 
![image](https://user-images.githubusercontent.com/107632780/229260568-36cdbe50-1861-45b4-8c5c-c6231ae50c88.png)

- - You can always as for guidance by leaving a comment under this pull request or reaching out to your peers or mentors on slack. 😄 😄